server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -0,0 +1,77 @@
+package com.objectcomputing.checkins.services.skills;
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@Entity
+@Table(name ="skills")
+public class Skill {
+
+    public Skill() {
+    }
+
+    public Skill(String name) {
+        this(name, true);
+    }
+
+    public Skill(String name, boolean pending) {
+        this.name = name;
+        this.pending = pending;
+    }
+
+    @Id
+    @Column(name="skillid")
+    @AutoPopulated
+    @TypeDef(type= DataType.STRING)
+    private UUID skillid;
+
+    @NotBlank
+    @NotNull
+    @Column(name="name", nullable = true, unique = true)
```

Comment:
we're using @NotNull and nullable=true at the same time. Is this a good practice?